### PR TITLE
[4.4] com_newsfeeds deprecated strings

### PR DESCRIPTION
**Pull request by brianteeman**
Marks some old legacy strings as deprecated

### CODE REVIEW

File: administrator/language/en-GB/com_newsfeeds.ini [hunk 1]
```diff
@@ -81,18 +81,23 @@ COM_NEWSFEEDS_N_QUICKICON_1="News Feed"
 COM_NEWSFEEDS_N_QUICKICON_SRONLY="News Feeds: %d news feeds are published."
 COM_NEWSFEEDS_N_QUICKICON_SRONLY_1="News Feeds: One news feed is published."
 COM_NEWSFEEDS_NEW_NEWSFEED="New News Feed"
-COM_NEWSFEEDS_NEWSFEEDS="News Feeds"
+;COM_NEWSFEEDS_NEWSFEEDS="News Feeds"
```
Comment (laoneo) [inline administrator/language/en-GB/com_newsfeeds.ini]: Is this change correct?

Comment (brianteeman) [reply administrator/language/en-GB/com_newsfeeds.ini]: no its a mistake from my tests

**Comment by laoneo** [comment]
Thanls!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**